iocage_lib/ioc_json.py

```diff
@@ -384,8 +386,8 @@ def validate_rctl_props(prop, value):
                     iocage_lib.ioc_common.logit(
                         {
                             'level': 'EXCEPTION',
-                            'message': 'b, k, m, g, t, p suffixes are not '
```

Comment:
I'm wondering if we can't just grab the last character of the value and print that here instead of a long list of every possible capitalization.

Ex: value[-1]

Reply:
Sounds reasonable.

Reply:
Fixed it.